### DISCUSSION TITLE
SmartAutoClicker-1.3.1

### DISCUSSION THREAD

**Discussion by Nain57** [Nain57/Smart-AutoClicker#113] [Announcements]
### Quick fix release:

Fix a bug preventing the touch event to go to the activity when the main menu was displayed. (https://github.com/Nain57/Smart-AutoClicker/issues/111)
Thanks to @zakius for the support !

<hr /><em>This discussion was created from the release <a href='https://github.com/Nain57/Smart-AutoClicker/releases/tag/1.3.1'>SmartAutoClicker-1.3.1</a>.</em>

**Comment by baskuza** [comment]
how to transfer marco to another device?

**Reply by Nain57** [reply]
This can't be done currently, maybe in the future ;)

**Reply by vdurante** [reply]
@Nain57 +1 on this feature! Would be amazing. I just bought a new phone and had to reset everything.

**Reply by Nain57** [reply]
This is planified for the next major release.
I just have to find a way to make the scenario works on any screens, which is not easy

**Reply by zakius** [reply]
I don't think it's even possible as interfaces may scale/behave in a non-trivial way, though still it would be valuable to be able to backup your scenarios to a file. And if someone happens to have multiple same model devices it's just an extra

**Comment by joyia543** [comment]
@Nain57 sir could you please add music or tone play on button click?
